SAMtags.tex

```diff
@@ -58,6 +58,7 @@ \section{Standard tags}
   {\tt BC} & Z & Barcode sequence \\
   {\tt BQ} & Z & Offset to base alignment quality (BAQ) \\
   {\tt CC} & Z & Reference name of the next hit \\
+  {\tt CG} & B,I & BAM-only tag to store the real {\sf CIGAR} if it contains $>$65535 operations\\
```

Comment:
I think that it should be allowed to leave the CG tag in a sam or a cram. the "recommended practice" can talk about the fact that only bam needs it, but there's no reason to complicate thing in the spec by making it illegal in anything but bam.